static/app/components/events/aiSuggestedSolution/experimentalFeatureBadge.tsx

```diff
@@ -9,9 +9,4 @@ export function ExperimentalFeatureBadge() {
 
 const CenteredFeatureBadge = styled(FeatureBadge)`
```

Comment:
The text was not centered before:
<img width="224" alt="Screenshot 2024-01-31 at 1 27 33 PM" src="https://github.com/getsentry/sentry/assets/30991498/0d67eb4d-2e81-41c5-81c8-1ed790e28fd9">
After:
<img width="209" alt="Screenshot 2024-01-31 at 1 27 52 PM" src="https://github.com/getsentry/sentry/assets/30991498/54e3d471-d844-4206-8782-0925b2125bcc">